testing/opae-c/test_error_c.cpp

```diff
@@ -68,10 +67,14 @@ class error_c_p : public ::testing::TestWithParam<std::string> {
   }
 
   virtual void TearDown() override {
-    EXPECT_EQ(fpgaDestroyProperties(&filter_), FPGA_OK);
+    if (filter_ != nullptr) {
+      EXPECT_EQ(fpgaDestroyProperties(&filter_), FPGA_OK);
+    }
     uint32_t i;
-    for (i = 0 ; i < num_matches_ ; ++i) {
-        EXPECT_EQ(fpgaDestroyToken(&tokens_[i]), FPGA_OK);
+    for (auto &t : tokens_) {
```

Comment:
You're not setting the individual tokens_ values back to nullptr.  Will this cause an issue with this approach?  That's one reason I used the num_matches_ approach - I knew exactly how many to destroy.